src/Compilers/CSharp/Test/Syntax/Parsing/LocalFunctionParsingTests.cs

```diff
@@ -2641,5 +2641,217 @@ void verify()
                 EOF();
             }
         }
+
+        [Fact]
+        public void ParameterScope_InMethodAttributeNameOf_AnonymousFunctionWithImplicitParameters()
+        {
```

Comment:
PROTOTYPE comment would be good here. This is a regression in parsing

Reply:
:)